plugins/woocommerce-blocks/assets/js/blocks/cart-checkout-shared/payment-methods/test/payment-methods.js

```diff
@@ -46,27 +46,30 @@ jest.mock( '@woocommerce/blocks-components', () => {
 	};
 } );
 
-const originalSelect = jest.requireActual( '@wordpress/data' ).select;
-const selectMock = jest
-	.spyOn( wpDataFunctions, 'select' )
-	.mockImplementation( ( storeName ) => {
-		const originalStore = originalSelect( storeName );
-		if ( storeName === PAYMENT_STORE_KEY ) {
-			return {
-				...originalStore,
-				getState: () => {
-					const originalState = originalStore.getState();
-					return {
-						...originalState,
-						savedPaymentMethods: {},
-						availablePaymentMethods: {},
-						paymentMethodsInitialized: true,
-					};
-				},
-			};
-		}
-		return originalStore;
-	} );
+jest.mock( '@wordpress/data', () => {
+	const originalModule = jest.requireActual( '@wordpress/data' );
+	return {
+		...originalModule,
+		select: jest.fn( ( storeName ) => {
+			const originalStore = originalModule.select( storeName );
+			if ( storeName === 'wc/store/payment' ) {
```

Comment:
I'm curious why this is changed from using constant `PAYMENT_STORE_KEY` to hardcoded string